04-Anatomy_of_ASDF_Internals/04-Anatomy_of_ASDF_Internals.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<p><code>print(tree)</code> instead of just <code>tree</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p><code>print(tree)</code> generates ugly output:   </p><pre class="ql-syntax" spellcheck="false">{'asdf_library': {'author': 'The ASDF Developers', 'homepage': 'http://github.com/asdf-format/asdf', 'name': 'asdf', 'version': '2.11.2.dev15+g6703d8f.d20220607'}, 'history': {'extensions': [{'extension_class': 'asdf.extension.BuiltinExtension', 'software': {'name': 'asdf', 'version': '2.11.2.dev15+g6703d8f.d20220607'}}]}, 'a': 6.0, 'b': True, 'c': 11, 'd': 'goodbye world', 'e': [1, 4, 9, 16], 'f': {'x': [1, 3, 5], 'y': {'nests': True, 'top': False}}}
</pre><p> </p>